make/ex_doc_wrapper.in

```diff
@@ -4,7 +4,7 @@ ARGS=("$@")
 
 set -eo pipefail {0}
 
-EX_DOC=$(command -v ex_doc)
+EX_DOC=$(command -v ex_doc || true)
```

Comment:
I am getting "Could not find ex_doc!" because this is overriding `EX_DOC`.

```
EX_DOC=${EX_DOC:=$(command -v ex_doc || true)}
```

Reply:
I just realized you are basically using `EX_DOC` to point to the wrapper.

Best case for the nix derivations is I can just define which EX_DOC to use directly and thread that into the wrapper.